package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A JavaScript-based private Tezos Blockchain Dev Tool that lives entirely on your local machine.",
   "main": "tezster.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha"
```

Comment:
YOu need to add a command for getting code-coverage.

Reply:
yeah I added "nyc" to that...

Reply:
I don't see any command in package.json, it should be like test-coverage or coverage
and you can run 'npm run test-coverage' to generate code coverage.